### ISSUE TITLE
Is it possible to do inference with TF 2.0?

### ISSUE THREAD

**Issue by ezorfa** [fizyr/keras-maskrcnn#93]
Hello,

I am making use of MaskRCNN in my project. My question is, at the moment, is it possible to do inference on TF 2.0 using current code of yours?

If not, by when are you planning to release code that is fully compatible with TF 2.0?

**Comment by vcarpani** [comment]
Hi, you can use this solution:

https://github.com/fizyr/keras-maskrcnn/issues/92

**Comment by SalahAdDin** [comment]
Is this implementation it fully usable?

**Comment by hgaiser** [comment]
It is, but tf2 is making it a bit tricky. Best bet is to use `master` from both `keras-retinanet` and `keras-maskrcnn` and tf 2.1.0 (as of writing).

**Comment by SalahAdDin** [comment]
@hgaiser I'm using `keras-retinanet` with **TensorFlow** `2.1`, no problem with it.

But I need also to test `Mask R-CNN^ and it could be gread if it is possible with **TensorFlow** `2.1` as the **Matterport**'s has also problems with the latest version too.

**Comment by hgaiser** [comment]
What is the error you are getting ?
